docs/migrate.rst

```diff
@@ -58,6 +58,11 @@ bulk-size metrics property is dropped
 
 Metrics records for bulk request don't contain the ``bulk-size`` property anymore. Please use the ``weight`` property instead and consider the ``unit`` property to interpret the value.
 
+--include-filters and --exclude-filters affect all operations
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Prior to 2.0.3, administrative tasks/operations (see :ref:`operations documentation<track_operations>`) were exempt from filtering and would run regardless of filtering.  ``--include-filters`` and ``--exclude-filters`` flags now can affect all operations in a track.  If you make use of include filters, it is advised to check that all desired operations are listed.
```

Comment:
The respective command line flags are not called `--include-filters` / `--exclude-filters` but rather `--include-tasks` and `--exclude-tasks`. Also, as the filters are referring to tasks, I'd start this sentence with "Prior to 2.0.3, administrative tasks (see ...)".

Reply:
fixed